packages/components/src/components/ScalarSidebar/ScalarSidebarGroup.vue

```diff
@@ -55,9 +54,9 @@ const { cx } = useBindCx()
         class="group/group-button"
         :aria-expanded="open"
         :indent="level"
-        :selected="selected"
-        :disabled="disabled"
-        :icon="icon"
+        :selected
```

Comment:
Is this always true?

Reply:
This is actually a trick @bruno showed me, it's the same as writing `:selected="selected"` 🤯

Reply:
alright this changes everything